### PR TITLE
README.md: add basic installation instructions for Ubuntu

### LINKED ISSUE #450: Add basic installation instructions.
Fulfilled by PR #449

I originally wrote the instructions here: https://github.com/rtyley/bfg-repo-cleaner/issues/255#issuecomment-1500711841

### PR DESCRIPTION
**Pull request by ElectricRCAircraftGuy**
Add basic installation instructions for Ubuntu, as a demonstration for even the most beginner users.

Note: these instructions were first posted here:
https://github.com/rtyley/bfg-repo-cleaner/issues/255#issuecomment-1500711841

Resolves #450

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,59 @@ filter-repo requires:
 
 `git-filter-repo` is a single-file python script, which was done to make
 installation for basic use on many systems trivial: just place that
-file into your $PATH.
+file into your `$PATH`. Example:
+
+### How to install `git filter-repo` on Linux Ubuntu:
```
Comment (newren) [inline README.md:51]: Thanks for the PR, but adding hundreds of sections ("How to install on Arch Linux", "How to install on Fedora Linux", "How to install on SuSE Linux", etc., then "How to install on Windows XP", "How to install on Windows 8", "How to install on Windows 11", then "How to install on Mac OS Ventura", "How to install on Mac OS Big Sur", etc., etc., etc.) makes absolutely no sense.  This isn't desirable.

File: README.md [hunk 2]
```diff
@@ -46,7 +46,59 @@ filter-repo requires:
...
+. ~/.profile
+
+# --------------------
+# 4. Done. Now run it.
+# --------------------
+git filter-repo --version       # check the version
+git filter-repo -h              # help menu
+git filter-repo -h | less -RFX  # help menu, viewed in the `less` viewer
+```
+
```
Comment (newren) [inline README.md:100]: None of this stuff belongs on the main page; the separate [install page](../../blob/main/INSTALL.md) is where more detailed explanations go.  I think that page already covers the necessary information from here, though.

**Comment by ElectricRCAircraftGuy** [comment]
@newren , would you be willing to activate the community wiki for this repo? Then I'd just paste my info there.

File: README.md (hunk 2 above)
Comment (ElectricRCAircraftGuy) [reply README.md:100]: I'm not going to win any argument with you, so I'm not going to try, but looking at your install page, I would actually say your install page doesn't include *any* of the information I've included in this PR, except this:

> If you place the git-filter-repo script in your $PATH,

File: README.md (hunk 1 above)
Comment (ElectricRCAircraftGuy) [reply README.md:51]: @newren , would you be willing to activate the community wiki for this repo? Then I'd just paste my info there.

File: README.md (hunk 2 above)
Comment (ElectricRCAircraftGuy) [reply README.md:100]: I think the point about not wanting a ton of installation-related PRs is valid. Activating the community wiki for this repo would solve that.

**Comment by newren** [comment]
I'm going to close this out.  I think the Ubuntu instructions should just be `apt-get install git-filter-repo`, and that, as noted above, these instructions -- while they have good intentions -- is actually going to make it harder for most users and cause problems as per the many bullet points I listed above.

**Pull request closed without merging**